### PR TITLE
Use DID JSON-LD Context from Candidate Recommendation stage.

### PR DESCRIPTION
**Pull request by msporny**
Updates DID Core context to:

1. Remove non-DID Core properties.
2. Enable term protection.

### CODE REVIEW

**Review by dmitrizagidulin — APPROVED** [review]
:+1:

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "did-context",
-  "version": "2.0.1-0",
+  "version": "3.0.0",
```
Comment (mattcollier) [inline package.json]: this line needs to be backed out, the publish script handles this.

**Review by mattcollier — CHANGES_REQUESTED** [review]
TY, also needs a changelog entry.

File: package.json (hunk 1 above)
Comment (dmitrizagidulin) [reply package.json]: Fixed.

**Comment by dmitrizagidulin** [comment]
@mattcollier I rolled back the version change, and added a CHANGELOG entry.

**Review by mattcollier — APPROVED** [review]
TY.

**Pull request merged**